### PR TITLE
migration for insertion of metacartel guild

### PR DESCRIPTION
**Pull request by mquellhorst**
Made a migration file for the guild table to insert the MetaCartel data 

Addresses #614

### CODE REVIEW

**Comment by dysbulic** [comment]
I'm somewhat confused. There is an existing [MetaCartel entry on MyMeta](https://my.metagame.wtf/guild/metacartel). If this is an update, shouldn't it be an `update` rather than an `insert`?

Also, it is likely prudent to include a `delete from guild where guildname = "metacartel"` in the down migration so that if, for some reason, it has to be reverted or rerun, it will succeed.

**Comment by dysbulic** [comment]
Ok, is MetaCartel Ventures a different organization than MetaCartel?

¿Did this migration succeed? It seems like it should have failed with a conflict on `guildname`.

**Review by alalonde — CHANGES_REQUESTED** [review]
This is for MetaCartel Ventures, which has its own DAO. See https://metacartel.xyz/ 

`guildname` must be unique. I believe `metacartel` already exists. You can query the prod mymeta API (api.metagame.wtf) to view the existing guild information.

Also, please reference the original issue (e.g. #issue-num) when creating PRs. Any additional guild information can be requested in that issue.

**Comment by mquellhorst** [comment]
Ah ok I got the idea it was about MetaCartel but you're right, that's already on there and it needs to be MetaCartel Ventures. I'll change the migration and ref the original issue

**Comment by alalonde** [comment]
Did you do a merge or rebase? If done properly, this PR should only show your 2 commits.

File: hasura/metadata/tables.yaml [hunk 1]
```diff
@@ -315,7 +315,6 @@
       - timezone
       - username
       filter: {}
-      allow_aggregations: true
```
Comment (alalonde) [inline hasura/metadata/tables.yaml:318]: I don't think you want to do this? Looks like a bad merge

**Review by dan13ram — APPROVED** [review]

File: hasura/metadata/tables.yaml (hunk 1 above)
Comment (mquellhorst) [reply hasura/metadata/tables.yaml]: Fixed it :)

**Review by alalonde — APPROVED** [review]
Thanks!

**Pull request merged**